src/Logic/operate.js

```diff
@@ -0,0 +1,31 @@
+import Big from 'big.js';
+
+let result = Infinity;
+
+const Operate = (numberOne = null, numberTwo = null, operation) => {
+  const operations = ['+', '-', 'X', '/', '%'];
+  if (operations.includes(operation)) {
+    if (operation === '+') {
+      result = new Big(numberOne).plus(new Big(numberTwo));
+    } else if (operation === '-') {
+      result = new Big(numberOne).minus(new Big(numberTwo));
+    } else if (operation === 'X') {
+      result = new Big(numberOne).times(new Big(numberTwo));
+    } else if (operation === '/') {
+      try {
+        result = new Big(numberOne).div(new Big(numberTwo));
+      } catch (error) {
```

Comment:
# [Optional]
Your code appears to be WET-(Write Every Time), Microverse encourage DRY-(Don't Repeat Yourself) code.
In your case, you repeated `new Big`.
You could have it in a variable then reuse it.
example:
```
    const firstNum = Big(numberOne);
    const secondNum = Big(numberTwo);
```

Reply:
Done